t/sdam_spec.t

```diff
@@ -28,6 +28,7 @@ my $iterator = path('t/data/SDAM')->iterator({recurse => 1});
 
 while ( my $path = $iterator->() ) {
     next unless -f $path && $path =~ /\.json$/;
+    #next unless $path =~ /too_old/;
```

Comment:
Stray comment.  :-)

But in the case of the /too_old/ test, we should just fiddle the compatible result to 'true' somewhere later in this test runner so we can make sure it actually works.